.gitlab/helper_functions.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+CYAN="\e[0;36m"
+CLEAR="\e[0m"
+SECTION_START="\e[0Ksection_start:the_time:section_id\r\e[0K${CYAN}section_header${CLEAR}"
+SECTION_END="\e[0Ksection_end:the_time:section_id\r\e[0K"
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+section_start() {
+    local section_header section_id start
+    start="$SECTION_START"
+    if [[ "$#" -eq 1 ]]; then
+        section_header="$1"
+        section_id="$(echo "$1" | tr -c '[:alnum:]\n\r' '_')"
```

Comment:
`\n\r` or `\r\n` ? (i.e. `CRLF` or `LFCR`?)
In several places.

Reply:
the order is not important here in `tr`